haystack/nodes/reader/table.py

```diff
@@ -574,14 +606,20 @@ def predict(self, query: str, documents: List[Document], top_k: int) -> Dict:
                 no_answer_score = current_no_answer_score
 
         if self.return_no_answer:
+            if self.return_table_cell:
+                offsets_in_context = None
+                offsets_in_document = None
+            else:
+                offsets_in_context = [Span(start=0, end=0)]
+                offsets_in_document = [Span(start=0, end=0)]
```

Comment:
I'm not 100% on this, but I think point 2 is correct. I believe we used to check for `Span(start=0, end=0)` to check for No Answers, but I don't think we do this any longer. I'll go ahead and make the change to `None`.

Reply:
So I did try switching to `None` instead of `Span` and that didn't cause any tests to fail. 

However, thinking more on this, changing this to `None` could cause breaking changes in downstream end-user tasks which I think we should avoid. Once we deprecate `Span` we can safely switch to only returning `None` since that will be the behavior for `TableCell`. What do you think?